sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -282,9 +282,9 @@ def check_identifier_presence_and_update_mux_table_entry(state_db, port_tbl, y_c
     else:
         # Convert list of tuples to a dictionary
         mux_table_dict = dict(fvs)
-        if "mux_cable" in mux_table_dict:
-            val = mux_table_dict.get("mux_cable", None)
-            if val == "true":
+        if "state" in mux_table_dict:
+            val = mux_table_dict.get("state", None)
+            if val == "active" or val == "auto":
```

Comment:
This seems unnecessary, `val` will always be either `active` or `auto`

Reply:
It can't hurt to confirm. However, I suggest simplifying and making more Pythonic:

```python
if val in ["active", "auto"]:
```

Reply:
fixed according to Joe's suggestion